packages/core/strapi/lib/services/entity-validator/index.js

```diff
@@ -182,7 +198,10 @@ const createValidateEntity = createOrUpdate => async (model, data, { isDraft = f
     );
   }
 
-  const validator = createModelValidator(createOrUpdate)(model, data, { isDraft }).required();
+  const validator = createModelValidator(createOrUpdate)(model, data, {
```

Comment:
To make things a bit easier maybe we could pass down model / entity / data instead of entityId or uid ? this would make this simpler while giving more flexibility ?